catt/cli.py

```diff
@@ -206,23 +179,32 @@ def cast(settings, video_url, subtitle, force_default, random_play, no_subs, no_
         video_id = stream.video_id or stream.playlist_all_ids[0]
         cst.play_playlist(stream.playlist_id, video_id=video_id)
     else:
+        if not subtitles and not no_subs and stream.is_local_file:
+            subtitles = hunt_subtitles(video_url)
+        if subtitles:
+            subs = SubsInfo(subtitles, stream.local_ip, stream.port + 1)
+            su_thr = Thread(target=serve_file, args=(subs.file, stream.local_ip, subs.port, "text/vtt;charset=utf-8"))
```

Comment:
We might want to abstract these three lines into a function, so we don't repeat them every time.